RecoBTag/Combined/python/caloDeepCSVJetTags_cfi.py

```diff
@@ -0,0 +1,24 @@
+import FWCore.ParameterSet.Config as cms
+
+caloDeepCSVJetTags = cms.EDProducer(
+	'DeepFlavourJetTagsProducer',
+	src = cms.InputTag('caloDeepCSVTagInfos'),
+  checkSVForDefaults = cms.bool(False),
+  meanPadding = cms.bool(False),
+	NNConfig = cms.FileInPath('RecoBTag/Combined/data/DeepFlavourNoSL.json'),
+  toAdd = cms.PSet(
+      probcc = cms.string('probc')
+      ),
+	)
+
+from Configuration.Eras.Modifier_phase1Pixel_cff import phase1Pixel
+phase1Pixel.toModify(caloDeepCSVJetTags, NNConfig = cms.FileInPath('RecoBTag/Combined/data/DeepCSV_PhaseI.json'))
+phase1Pixel.toModify(caloDeepCSVJetTags, checkSVForDefaults = cms.bool(True))
+phase1Pixel.toModify(caloDeepCSVJetTags, meanPadding = cms.bool(True))
+phase1Pixel.toModify(caloDeepCSVJetTags, toAdd = cms.PSet())
+
+from Configuration.Eras.Modifier_phase2_common_cff import phase2_common
+phase2_common.toModify(caloDeepCSVJetTags, NNConfig = cms.FileInPath('RecoBTag/Combined/data/DeepCSV_PhaseI.json'))
+phase2_common.toModify(caloDeepCSVJetTags, checkSVForDefaults = cms.bool(True))
+phase2_common.toModify(caloDeepCSVJetTags, meanPadding = cms.bool(True))
+phase2_common.toModify(caloDeepCSVJetTags, toAdd = cms.PSet())
```

Comment:
@slava77 
I have cloned pf based with edit of src value as follows:
[https://gist.github.com/tiwariPC/5c2f2fec42afda7389f8f884814caf6a](url)

Additionally above lines also contain `caloDeepCSVJetTags`, can you provide your feedback on what should be the proper solution for this(if it is not okay)? 

I have also updated other two files.
Please let me know if there is any issue.